ui/js/compareperf.js

```diff
@@ -5,12 +5,21 @@
 "use strict";
 
 perf.controller('CompareChooserCtrl', [
-  '$state', '$stateParams', '$scope', 'ThRepositoryModel', 'ThResultSetModel',
+  '$state', '$stateParams', '$scope', 'ThRepositoryModel', 'thDefaultRepo', 'ThResultSetModel',
   function CompareChooserCtrl($state, $stateParams, $scope,
-                              ThRepositoryModel, ThResultSetModel) {
+                              ThRepositoryModel, thDefaultRepo, ThResultSetModel) {
     ThRepositoryModel.get_list().success(function(projects) {
       $scope.projects = projects;
-      $scope.originalProject = $scope.newProject = projects[0];
+      $scope.originalProject = $scope.newProject = getDefaultRepo();
```

Comment:
Ok, so for this one let's define two additional variables in `treeherder/ui/js/values.js`:

    treeherder.value(phCompareDefaultOriginalRepo, 'mozilla-central');
    treeherder.valueu(phCompareDefaultNewRepo, 'try');

Then try to set the values of originalProject and newProject here to that appropriately. I'll leave the rest of the details to you to figure out. :)